pkg/manifest/manifest_test.go

```diff
@@ -587,39 +587,62 @@ func setupTestFS(t *testing.T, d dir) (string, func() error) {
 }
 
 func Test_include(t *testing.T) {
+	identifier := "identifier"
+	defaultClusterProfile := "self-managed-high-availability"
+	singleNodeProfile := "single-node"
+	trueBol := true
+	falseBol := false
+
 	tests := []struct {
 		name               string
-		exclude            string
-		includeTechPreview bool
-		profile            string
+		exclude            *string
+		includeTechPreview *bool
+		profile            *string
 		annotations        map[string]interface{}
-		caps               configv1.ClusterVersionCapabilitiesStatus
+		caps               *configv1.ClusterVersionCapabilitiesStatus
 
 		expected error
 	}{
 		{
 			name:    "exclusion identifier set",
-			exclude: "identifier",
-			profile: DefaultClusterProfile,
+			exclude: &identifier,
+			profile: &defaultClusterProfile,
+			annotations: map[string]interface{}{
+				"exclude.release.openshift.io/identifier":                     "true",
+				"include.release.openshift.io/self-managed-high-availability": "true"},
+			expected: fmt.Errorf("exclude.release.openshift.io/identifier=true"),
+		},
+		{
+			name:    "exclusion identifier set with no capability",
+			exclude: &identifier,
+			profile: &defaultClusterProfile,
 			annotations: map[string]interface{}{
 				"exclude.release.openshift.io/identifier":                     "true",
 				"include.release.openshift.io/self-managed-high-availability": "true"},
+			caps:     &configv1.ClusterVersionCapabilitiesStatus{},
 			expected: fmt.Errorf("exclude.release.openshift.io/identifier=true"),
 		},
 		{
 			name:        "profile selection works",
-			profile:     "single-node",
+			profile:     &singleNodeProfile,
 			annotations: map[string]interface{}{"include.release.openshift.io/self-managed-high-availability": "true"},
 			expected:    fmt.Errorf("include.release.openshift.io/single-node unset"),
 		},
+		{
+			name:        "No profile",
```

Comment:
Change this to "all nil", which I think it is, and cover all nil's with one case. Or add a case for each one set to nil.

Reply:
I will add an all nil use case which will be equivalent to no annotation test case. The include method reacts differently when each of the input is nil so I have handled each use case differently.

Reply:
Fixed in https://github.com/openshift/library-go/commit/7ae641e6db34e5fad1456c5954b03f2b51292cae